configs/tasks/obj_nav_mp3d.yaml

```diff
@@ -0,0 +1,47 @@
+ENVIRONMENT:
+  MAX_EPISODE_STEPS: 750
```

Comment:
Why 750?

Reply:
As I remember, to give an option to explore the environment and then choose an object.